### DISCUSSION TITLE
Removing the zitadel.{$DOMAIN} or disabling subdomain per org

### DISCUSSION THREAD

**Discussion by idc77** [zitadel/zitadel#8756] [Q&A]
I have now somehow setup zitadel with nginx on the front and external secure on a subdomain of my primary domain.

id.domain.tld

When logging in I had to adjust the user to ` zitadel-admin@zitadel.id.domain.tld` instead of localhost (some people might find this useful, because it's not mentioned anywhere).

So zitadel now believes it's on the zitadel.id.domain.tld, despite it being under id.domain.tld .

When I go to default setup, the custom domains list is
- zitadel-0figc3.id.domain.tld
- id.domain.tld

I don't have a wildcard dns set in my dns configuration and don't intend to.
I don't even want this functionality of subdomain per org.

Is it possible to disable this?

Also, can I safely rename the initial ZITADEL org?

**Comment by hifabienne** [comment]
@eliobischof can you help here?

**Comment by idc77** [comment]
- Also, can I safely rename the initial ZITADEL org?
Yes, I renamed it to default

- Is it possible to disable this?
It seems to work like keycloak realms
I have created a wildcard tls cert and pointed *.id.domain.tld to the IP, but then when going to one of those subdomain, since ORIGIN is set, it won't continue and shows that, origin doesn't match the current domain.

It feels like a puzzle piece that doesn't want to fit, not matter what you do, something is always wrong.
Maybe it makes sense for your cloud offering, but then I'd like to see documented how you configure the origin

I'd also like to remove the :443 from the issuer URL.

**Reply by m-urban** [reply]
I have the exact same setup and question. In reference to @idc77's example – how can I remove zitadel-0figc3.id.domain.tld from my configuration?

**Reply by idc77** [reply]
@eliobischof apparently you're the person to ping so, sorry if I do, but there has been no progress in almost a month regarding this. Can ORIGIN be set to a wildcard subdomain? Is it possible to remove the :443 suffix?
